indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -526,6 +543,22 @@ class BookingDetails extends React.Component {
               <Translate>Booking Details</Translate>
             </span>
             <span styleName="booking-status">{this.renderBookingStatus()}</span>
+            <span styleName="clipboard-copy">
```

Comment:
I think you can just set the style directly on the `Icon` component (and this particular style is something I'd set using the `style` prop instead of a CSS file)